### ISSUE TITLE
Installing extension in PostgreSQL 9.6

### ISSUE THREAD

**Issue by dmagrinya** [postgrespro/jsquery#25]
Hi, is it possible to install the extension to a system running PostgreSQL 9.6? I'm trying to do it on a system where I have 'bison' version 2.7 installed but it prints the following error after running the command `$ make USE_PGXS=1`:

***
ERROR: bison is missing on your system. It is needed to create the
file `jsquery_gram.c'. You can either get bison from a GNU mirror site
or download an official distribution of PostgreSQL, which contains
pre-packaged bison output.
***

**Comment by robe2** [comment]
I get the same error too on PostgreSQL 9.6 and PostgreSQL 10.  (compiling under Mingw64 gcc 4.9.2 where I've got BISON 3.0.4 installed.

Compiles fine against PostgreSQL 9.4 and 9.5.  So I assume it must be something included via PostgreSQL 9.6 and 10 tree causing the issue.

I was able to work around the issue by using  the jsquery_gram.c built by my PostgreSQL 9.5 compile.  Seems to work fine in 9.6, but that's not ideal and not sure if it will cause problems down the line.
